rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/nodesource/NSCreationServlet.java

```diff
@@ -271,6 +248,21 @@ private void createNs(HttpServletRequest request, HttpServletResponse response)
         }
     }
 
+    private void extractHiddenItems(Map<String, String> infraParams, Map<String, String> infraFileParams,
+            String formFieldValue) {
+        for (String row : formFieldValue.split(ROW_SEPARATOR)) {
+            String[] fields = row.split(FIELD_SEPARATOR);
```

Comment:
Might we need to check `fields` length ?

Reply:
ok thanks